package.json

```diff
@@ -10,6 +10,7 @@
     "eslint": "6.6.0",
     "eslint-config-prettier": "6.4.0",
     "eslint-plugin-babel": "5.3.0",
+    "eslint-plugin-import": "^2.20.2",
```

Comment:
Please use a strict version for dependency, and add a loose one to `peerDependencies`